suse-openstack-cloud-rpm-list

```diff
@@ -76,36 +89,50 @@ openstack-heat-doc
 openstack-heat-engine
 openstack-heat-plugin-heat_docker
 openstack-heat-templates
+openstack-heat-test
```

Comment:
It's in the tempest category as `<package name="openstack-heat-test" supportstatus="unsupported"/>`.